src/components/Search.tsx

```diff
@@ -1,18 +1,25 @@
 import React from 'react'
-import { View, StyleSheet, TextInput } from 'react-native'
+import { View, StyleSheet, TextInput, Platform } from 'react-native'
 import Icon from 'react-native-vector-icons/MaterialCommunityIcons'
 
 const styles = StyleSheet.create( {
   searchBar: {
-    padding: 10,
+    paddingLeft: Platform.OS === 'android' ? 10 : undefined,
+    padding: Platform.OS === 'ios' ? 10 : undefined,
     flexDirection: 'row',
+    borderRadius: 10,
+    backgroundColor: 'rgb(220,219,224)',
```

Comment:
I wish we had a lint rule for this, but consistent formatting in terms of spaces after comma, and/or using Hex values. Thoughts, @bhajneet?

This will change as per #115.

Reply:
I don't care to normalise rgba and hex, but more so that colors should be referenced only as  variables.

Reply:
Spaces after comma within rgba would be nice, not sure if there is a rule for that

Reply:
For now should I move this to a `src/theme/colors.js`?

Reply:
Once there are a few components (Which I think there are now?) it would be a good idea to start consolidating colors somewhere, yes. Path/Location is up to you guys.